tests/ui/await_holding_lock.rs

```diff
@@ -1,64 +1,194 @@
 #![warn(clippy::await_holding_lock)]
 
-use std::sync::Mutex;
+extern crate parking_lot;
```

Comment:
Is this needed because we're in a UI test? Elsewhere I'd just `use`.

Reply:
Not sure. I just copied this from the `let_underscore` test, that also tests for the parking_lot crate. I'll try if it also works with `use`, now that we're using the 2018/2021 (?) edition.